week2/index.js

```diff
@@ -1,65 +1,129 @@
+//Include File File System package.
 var fs = require('fs');
 
+//Get input commands, remove 'node index.js' form the command.
 var options = process.argv.slice(2);
 
-var command = options[0];
-
-switch (command) {
-  case 'help':
-  default:
-    showHelp();
-    break;
-  case 'list':
-    listTodos();
-    break;
+// find the value of the command, call help file in case command value not found.
+switch (options[0]) {
+	case 'help':
+	default:
+		showHelp();
+		break;
+	case 'list':
+		listTodos();
+		break;
+	case 'add':
+		addLine();
+		break;
+	case 'remove':
+		deleteLine();
+		break;
+	case 'reset':
+		resetFile();
+		break;
+	case 'update':
+		updateLine();
+		break;
 }
-
+//separate by new line file content, and filter it in array.
 function splitStringByNewline(string) {
```

Comment:
Good use of what was built in - for an extra challenge, how could you convert this to read from/write to a JSON file?